pycbc/inference/models/tools.py

```diff
@@ -766,7 +770,12 @@ def get_loglr():
             self.reconstruct_phase = False
 
         rec['loglr'] = loglr[xl]
-        rec['loglikelihood'] = self.lognl + rec['loglr']
+        if set_others_lognl is None:
```

Comment:
This could probably be handled in your model since lognl doesn't affect the drawing of anything (it's a constant in your model), no?

Reply:
That would mean you wouldn not need to pass this in, just update it in your method.